ansible_3par_docker_plugin/uninstall/uninstall_hpe_3par_volume_driver.yml

```diff
@@ -15,12 +15,14 @@
       include: remove_multipath.yml
 
 - name: Remove HPE 3PAR Volume Driver for Kubernetes/OpenShift
-  hosts: masters
+  hosts: "{{groups['masters'][0]}}"
```

Comment:
Same needs to be done at https://github.com/hpe-storage/python-hpedockerplugin/blob/master/ansible_3par_docker_plugin/uninstall/uninstall_hpe_3par_volume_driver_etcd.yml#L22